python/ray/util/sgd/torch/examples/transformers/transformers_example.py

```diff
@@ -0,0 +1,370 @@
+# coding=utf-8
+# This is a modified example originally from The Google AI Language Team
```

Comment:
I think you have to specify the person/group who modified the code (as per Apache, IANAL tho)

Reply:
done.